lib/convert.js

```diff
@@ -60,7 +60,9 @@ var sdk = require('postman-collection'),
         requestNameSource: (options.hasOwnProperty('requestNameSource') ? options.requestNameSource : 'fallback'),
         schemaFaker: (options.hasOwnProperty('schemaFaker') ? options.schemaFaker : true),
         indentCharacter: (options.hasOwnProperty('indentCharacter') ? options.indentCharacter : ' '),
-        collapseLongFolders: (options.hasOwnProperty('collapseLongFolders') ? options.collapseLongFolders : true)
+        collapseLongFolders: (options.hasOwnProperty('collapseLongFolders') ? options.collapseLongFolders : true),
```

Comment:
@Dhroov7 The defaults are hard-coded here again. This should come directly from the getOptions function